### DISCUSSION TITLE
a doubt about onnx runtime installation steps

### DISCUSSION THREAD

**Discussion by irhallac** [open-mmlab/mmdeploy#1610] [Q&A]
Do we need to do both pip install and install from downloaded tgz at the same time? I see in Get Started/Step-1/ Linux-x86_64, CPU, ONNX Runtime 1.8.1 installation:
`pip install onnxruntime==1.8.1`
`wget https://github.com/microsoft/onnxruntime/releases/download/v1.8.1/onnxruntime-linux-x64-1.8.1.tgz`
`tar -zxvf onnxruntime-linux-x64-1.8.1.tgz`

**Comment by lvhan028** [comment]
`pip install onnxruntime` is for using onnxruntime's python API to perform onnx model inference.
`wget onnxruntime` uses onnxruntime's C++ API to perform SDK model inference.
By SDK model, it includes onnx model as well as some meta info that describes the model's preprocessing and postprocessing. 
The SDK model can be exported by `tools/deploy.py` with enabling `--dump-info`
